### PR TITLE
add default constructor for serialization libraries

### LINKED ISSUE #175: ListForLoopStatement cannot be transformed to json (no default constructor)

### PR DESCRIPTION
**Pull request by Bonczidai**
fixes #175 

Signed-off-by: Bonczidai Levente <levente.bonczidai@hp.com>

### CODE REVIEW

**Comment by Bonczidai** [comment]
@orius123 please review

File: cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ForLoopStatement.java [hunk 1]
```diff
@@ -15,6 +15,8 @@
 
 public abstract class ForLoopStatement implements Serializable{
 
+    private static final long serialVersionUID = -1467375536631244911L;
+
```
Comment (orius123) [inline cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ForLoopStatement.java]: why?

**Comment by orius123** [comment]
I added a test class.
can you add tests for the new classes?
just to be sure..

File: cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ForLoopStatement.java (hunk 1 above)
Comment (Bonczidai) [inline cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ForLoopStatement.java]: why WHY? :)

Comment (meirwah) [inline cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ForLoopStatement.java]: i think he means why you need serialVersionUID?

Comment (Bonczidai) [inline cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ForLoopStatement.java]: I thought it is best practice to generate serialVersionUID.. and I saw it is generated in other classes also (e.g. Input or InOutParam).. Should I remove them?

**Comment by Bonczidai** [comment]
added tests..

**Pull request merged**